src/components/Expandable.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react'
+import { Collapse } from 'react-bootstrap'
+import { BsChevronDown, BsChevronRight } from 'react-icons/bs'
+
+export const Expandable: React.FunctionComponent<{
+  id: string
+  header: string
+  defaultState?: boolean
```

Comment:
can you use a default value here? I.e. defaultState = false

I'm not sure if it works in template types.

Reply:
I don't think so with template types.